Task5Test/src/test/java/com/company/TriangleTest.java

```diff
@@ -0,0 +1,38 @@
+package com.company;
+
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class TriangleTest {

```

Comment:
Is that it? You must be joking. Please add the tests for all triangles types and positive/negative scenarious